+p_xml: 'version="1.0" encoding="UTF-8" standalone="yes" '
+directive: DOCTYPE bugzilla SYSTEM "http://bugzilla.fieldtriptoolbox.org/page.cgi?id=bugzilla.dtd"
bugzilla:
  +@version: 4.4.1
  +@urlbase: http://bugzilla.fieldtriptoolbox.org/
  +@maintainer: r.oostenveld@donders.ru.nl
  bug:
    bug_id: "1588"
    creation_ts: 2012-07-03 12:26:00 +0200
    short_desc: ft_fetch_sens should fetch units
    delta_ts: 2012-08-22 17:31:48 +0200
    reporter_accessible: "1"
    cclist_accessible: "1"
    classification_id: "1"
    classification: Unclassified
    product: FieldTrip
    component: core
    version: unspecified
    rep_platform: PC
    op_sys: Windows
    bug_status: CLOSED
    resolution: FIXED
    bug_file_loc:
    status_whiteboard:
    keywords:
    priority: P3
    bug_severity: normal
    target_milestone: '---'
    everconfirmed: "1"
    reporter:
      +content: johanna.zumer
      +@name: Johanna
    assigned_to:
      +content: johanna.zumer
      +@name: Johanna
    comment_sort_order: oldest_to_newest
    long_desc:
      - +@isprivate: "0"
        commentid: "6665"
        comment_count: "0"
        who:
          +content: johanna.zumer
          +@name: Johanna
        bug_when: 2012-07-03 12:26:19 +0200
        thetext: I had an error in my leadfield computation due to sens and vol having different units.  I assumed FT would either convert for me, or error. (Indeed ft_prepare_vol_sens does have a check for compatible units).  However, the call to ft_fetch_sens in prepare_headmodel drops the units of sens, thus the unit check in ft_prepare_vol_sens didn't complain because sens no longer had units.  I asked Joern, and he said go ahead and add the units if they exist, so I will.
      - +@isprivate: "0"
        commentid: "6666"
        comment_count: "1"
        who:
          +content: johanna.zumer
          +@name: Johanna
        bug_when: 2012-07-03 13:16:52 +0200
        thetext: fixed, svn commit 6210.
